src/libraries/Common/tests/Extensions/TestingUtils/Microsoft.AspNetCore.Testing/test/ConditionalFactTest.cs

```diff
@@ -48,12 +48,14 @@ public void ThisTestMustRunOnCLR()
 #error Target frameworks need to be updated.
 #endif
 
+#if !SKIP_FAILING
```

Comment:
Same here, we dont need to test ConditionFactTest (unrelated to DI.tests). This is only included because DI.Tests uses some classes under TestingUtils/Microsoft.AspNetCore.Testing